programming_fundamentals/python_part_3/requirements.txt

```diff
@@ -3,3 +3,4 @@ netmiko>=3.3.2
 pyyaml>=5.3.1
 requests>=2.25.1
 xmltodict>=0.12.0
+pysnmp>=4.4.12
```

Comment:
@hpreston I had to add pysnmp to the requirements.txt for the GitHub Actions check to pass and for it to run locally for me - can you make sure this is correct?

Reply:
@annegentle  yeah that's right.  I left if off because that particular script/library isn't covered in the video.  But as the script is in the repo worth including the requirement.